runner/orchestration/build-serve-loop.ts

```diff
@@ -127,62 +128,109 @@ export async function attemptBuild(
       enableAutoCsp,
       userJourneyAgentTaskInput,
     );
+    if ('testCommand' in env) {
```

Comment:
Super nit (to be consistent with other code):

```suggestion
    if (env.testCommand !== undefined) {
```

Reply:
Probably also need to guard on build success here?

Reply:
`env.testCommand !== undefined` doesn't work because `testCommand` isn't in `BaseEnvironment`.

> Probably also need to guard on build success here?

There's a `break` guard above that guards both this and the Axe test

Reply:
Interesting! is there a reason it can't go into `BaseEnvironment`? Seems like a useful feature for remote environments too